doc/mgr/dashboard.rst

```diff
@@ -18,17 +18,17 @@ one and adds a built-in web based monitoring and administration application to
 the Ceph Manager. The architecture and functionality of this new plugin is
 derived from and inspired by the `openATTIC Ceph management and monitoring tool
 <https://openattic.org/>`_. The development is actively driven by the team
-behind openATTIC at `SUSE <https://www.suse.com/>`_.
-
-The intention is to reuse as much of the existing openATTIC functionality as
-possible, while adapting it to the different environment. openATTIC is based on
-Django and the Django REST Framework, whereas the dashboard plugin's backend
-code uses the CherryPy framework and a custom REST API implementation. The WebUI
-implementation is based on Angular/TypeScript, merging both functionality from
-the original dashboard as well as adding new functionality originally developed
-for the standalone version of openATTIC. The Ceph Manager Dashboard plugin is
-implemented as a web application that visualizes information and statistics about
-the Ceph cluster using a web server hosted by ``ceph-mgr``.
+behind openATTIC at `SUSE <https://www.suse.com/>`_, with a lot of support from
+the Ceph community and other companies like Red Hat.
+
+
+The dashboard plugin's backend code uses the CherryPy framework and a custom
```

Comment:
s/dashboard/Dashboard/ ?

Reply:
I don't think so (and this is consistent with the rest of the document) - while "Ceph Manager Dashboard" could be considered a "proper noun" (e.g. a brand or product name, which is supposed to be capitalized, see https://www.learnenglish.de/grammar/capitaltext.html), the term "dashboard" by itself is more generic and should thus not be written in capital letters.